finger/templates/mail_members_test.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h1>Mail our members</h1>
+
+{% if test %}
```

Comment:
When is this true?

Reply:
Detta är sant när man gör ett test utskick /mail_members (vilket händer när du kommer in på sidan första gången).